lib/tokens.js

```diff
@@ -80,7 +86,8 @@ bedrock.events.on('bedrock-mongodb.ready', async () => {
     options: {unique: true, background: false}
   }]);
 
-  // FIXME: create `tokenBBatch.id` shard key
+  // FIXME: create `tokenBatch.id` shard key on `tokenBatch` and
```

Comment:
does this imply that we hope to implement sharding in code here?  I don't know if that's a thing and if it is if it's something we would want to do at this level.

Reply:
@mattcollier yes, it implies that hope -- I also don't know if it's a thing.

Reply:
k, I'll create an issue to investigate the idea of sharding in code, if we can remove the fixme that would be great